lib/transport_web/templates/dataset/details.html.eex

```diff
@@ -103,27 +103,30 @@
       </div>
 
       <div class="dataset-details__validation">
-        <h2>Validations</h2>
+        <h1><%= dgettext("page-dataset-details", "GTFS Validation") %></h1>
         <%= for {issue_type, issues} <- @dataset.validations do %>
-          <h3><%= issue_type %></h3>
+        <label class="collapse" for="<%= issue_type %>">
+          <h3><%= Gettext.dgettext(TransportWeb.Gettext, "page-dataset-details", issue_type) %> (<%= Enum.count issues %>)</h3>
```

Comment:
> Gettext.dgettext

dgettext

Reply:
I don’t really know why, but it says it doesn’t work with dynamic strings, only with static. Hence the extra `Gettext`